### PR TITLE
Qe fix

### PR DESCRIPTION
**Pull request by mbaldessari**
- **Stop skipping the machine-sensor test**

### CODE REVIEW

File: tests/interop/run_tests.sh [hunk 1]
```diff
@@ -37,6 +37,10 @@ pytest -lv --disable-warnings test_check_logging_hub.py --kubeconfig $KUBECONFIG
 
 KUBECONFIG=$KUBECONFIG_EDGE pytest -lv --disable-warnings test_check_logging_edge.py --kubeconfig $KUBECONFIG_EDGE --junit-xml $WORKSPACE/test_check_logging_edge.xml
 
+# We need a few things from the hub cluster before working on the edge cluster
```
Comment (mlabonte-rh) [inline tests/interop/run_tests.sh:40]: We should probably define these in test_toggle_machine_sensor.py so that it works with internal CI too.

Comment (mlabonte-rh) [reply tests/interop/run_tests.sh:40]: We can't invoke 'oc' from a teflo (internal CI) definition file, so we will need to define these vars in test_toggle_machine_sensor.py anyway.
      
So, you could just remove lines 41-43 and replace it with:
export KUBECONFIG_HUB=$KUBECONFIG

and then we use KUBECONFIG_HUB to set the vars in test_toggle_machine_sensor.py

Does this make sense?

Comment (mbaldessari) [reply tests/interop/run_tests.sh:40]: I think so, thanks! I'll take a stab today

Comment (mbaldessari) [reply tests/interop/run_tests.sh]: Ok should be done, see if it is okay now?

**Pull request merged**